pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -776,6 +778,25 @@ func (bc *BuildPodBuilderConfig) unsupported() bool {
 	return bc.PlatformAPI != "0.2" && bc.PlatformAPI != "0.3"
 }
 
+func (bc *BuildPodBuilderConfig) tolerations() []corev1.Toleration {
+	tolerations := make([]corev1.Toleration, 0)
+
+	if bc.OS != "windows" {
```

Comment:
If taints are passed in do we care if it is windows/linux?

Reply:
no probably not